articles/post.php

```diff
@@ -4,71 +4,109 @@
+			$featured_image_position = get_post_meta( get_the_ID(), '_featured_image_position', true );
+
+			if ( ! $featured_image_position || sanitize_html_class( $featured_image_position ) !== $featured_image_position ) {
+				$featured_image_position = '';
+			}
+
+			?><figure class="featured-image <?php echo esc_attr( $featured_image_position ); ?>" style="background-image: url('<?php echo esc_url( $featured_image_src ); ?>');"><?php spine_the_featured_image(); ?></figure><?php
 		}
 		?>
+		</div>
 	</header>
 
-	<?php if ( ! is_singular() ) : ?>
-		<div class="article-summary">
+	<div class="article-body row side-right">
+		<div class="column one">
+			<?php the_content(); ?>
+		</div>
+		<div class="column two">
 			<?php
+			// Get a list of top level categories.
+			$categories = wp_get_object_terms( get_the_ID(), 'category', array(
+				'parent' => 0,
+			) );
 
-			if ( spine_has_thumbnail_image() ) {
-				?><figure class="article-thumbnail"><a href="<?php the_permalink(); ?>"><?php spine_the_thumbnail_image(); ?></a></figure><?php
-			} elseif ( spine_has_featured_image() ) {
-				?><figure class="article-thumbnail"><a href="<?php the_permalink(); ?>"><?php the_post_thumbnail( 'spine-thumbnail_size' ); ?></a></figure><?php
+			if ( 0 < count( $categories ) ) {
+				echo '<p class="meta-head meta-topic">Topics</p>';
+				echo '<ul class="meta-item-list">';
+
+				foreach ( $categories as $category ) {
+					$url = get_category_link( $category->term_id );
+					echo '<li class="meta-item"><a href="' . esc_url( $url ) . '">' . esc_html( $category->name ) . '</a>';
+				}
+
+				echo '</ul>';
 			}
 
-			if ( $subhead ) {
-				?><div class="article-subhead"><?php
-				echo wp_kses_post( apply_filters( 'the_content', $subhead ) );
-				?></div><?php
+			if ( function_exists( 'wsuwp_uc_get_object_people' ) ) {
+				$people = wsuwp_uc_get_object_people( get_the_ID() );
+			} else {
+				$people = array();
 			}
-			?>
 
-			<p class="article-read-more">
-				<a href="<?php echo esc_url( get_the_permalink() ); ?>">Read the article</a>
-			</p>
-		</div><!-- .article-summary -->
-	<?php else : ?>
-		<div class="article-body">
-			<?php the_content(); ?>
-			<?php
-			wp_link_pages( array(
-				'before' => '<div class="page-links">' . __( 'Pages:', 'spine' ),
-				'after' => '</div>',
-			) );
+			if ( 0 < count( $people ) ) {
+				echo '<p class="meta-head meta-people">People</p>';
+				echo '<ul class="meta-item-list">';
+
+				foreach ( $people as $person ) {
+					echo '<li class="meta-item"><a href="' . esc_url( $person['url'] ) . '">' . esc_html( $person['name'] ) . '</a></li>';
+				}
+
+				echo '</ul>';
+			}
 			?>
 		</div>
-	<?php endif; ?>
+	</div>
 
-	<footer class="article-footer">
+<?php } else { // If not is_single() ?>
+	<header class="article-header">
+		<h2 class="article-title">
+			<a href="<?php the_permalink(); ?>" rel="bookmark"><?php the_title(); ?></a>
+		</h2>
+	</header>
+<?php } ?>
+
+<?php if ( ! is_singular() ) : ?>
```

Comment:
I'm probably missing something, but it looks like the code in this condition could be consolidated under the `else` just above?